src/helm/benchmark/presentation/schema.py

```diff
@@ -214,6 +214,11 @@ class RunGroup(Field):
     # TODO: remove when we don't want helm-summarize to support runs before November 2023 anymore.
     adapter_keys_shown: List[str] = field(default_factory=lambda: ["model_deployment", "model"])
 
+    # Whether to accept non registered models or not.
+    # Used for huggingface models, a default ModelMetadata will be used when missing,
+    # Otherwise, if this is False, raises an error.
+    allow_unknown_models: bool = True
```

Comment:
Is there a reason why this should be configurable per-group? I feel like this should just be an field in `self.allow_unknown_models` in `Summarizer` that could be set by a flag on `summarize.py`.

Reply:
Ping.